test/NuGet.Core.Tests/NuGet.Protocol.Tests/DependencyInfoResourceV2FeedTests.cs

```diff
@@ -35,6 +36,7 @@ public class DependencyInfoResourceV2FeedTests
             // Act
             var dependencyInfoList = await dependencyInfoResource.ResolvePackages("WindowsAzure.Storage",
                                                                             NuGetFramework.Parse("aspnetcore50"),
+                                                                            It.IsAny<SourceCacheContext>(),
```

Comment:
This is fine, but again, a null cache context would be great for these scenarios, same as how the None token and NullLogger are used